README.md

```diff
@@ -0,0 +1,11 @@
+# Workplace Environmental Monitor Reference Deployment
+
+## Purpose
+
+This is an Arm Mbed reference deployment application of a workplace
+environmental monitor. The monitor contains environmental sensors for
+light, temperature and humidity. The sensor values are uploaded to Arm
+Mbed Cloud 1.2. The application also demonstrates firmware updates using
+the firmware-over-the-air (FOTA) capabilities of Mbed and Mbed Cloud. A
+plastic case contains the monitor. It is battery powered and has LED
```

Comment:
Suggested rephrasing: It is battery-powered, has LED indicators, and has an LCD display.